### ISSUE TITLE
No support for Printrbot Simple Metal

### ISSUE THREAD

**Issue by mosh1** [Printrbot/FirmwareUpdatr#4]
The updater doesn't have support for the Printrbot Simple Metal yet. This can cause users to get confused and flash the firmware for the Simple and end up in a bad state.

**Comment by PxT** [comment]
@abdrumm @lwalkera Which firmware versions should we provide options for in the app?  Let me know which hex files to link to and I can add them.

**Comment by mosh1** [comment]
According to: http://help.printrbot.com/Guide/Reflash+the+Firmware+with+OS+X/134

The firmware for Simple Metal 1403 is at: https://github.com/Printrbot/Marlin/releases/download/RevF-Version-1/RevF4-v1-SimpleMetal.hex

My friend was successfully able to flash to that version so it's confirmed to work.

**Comment by j-laird** [comment]
I was trying to build a version that would call an XML file that defined firmware and config files appropriate for a given model and rev of Printrboard.  Rather than hard coding these links into the software.

**Comment by PxT** [comment]
This thing is really just a Unix shell script wrapped in a simple GUI.  Getting it to parse XML may be an exercise in insanity…

A very basic config file format should be doable.  The current version only has one hex that it actually uses (Unified firmware) so it wasn't really an issue.

**Comment by PxT** [comment]
For example, a simple config could potentially look like:

```
# Model Name;firmware link;config link
Plus;http://example.com/firmware.hex;http://example.com/config.txt
Simple Metal;http://example.com/firmwareMetal.hex;http://example.com/configMetal.txt
```

Then that can be parsed inside the script fairly easily using standard Unix shell commands.

**Comment by fiveangle** [comment]
Confirmed that the hex linked by @mosh1 is correct for the 1403. I was "the friend" who flashed the incorrect using the Mac util and fixed by switching to Windows method (which is manual): resolved. 

I think adding the Printrboard Rev levels to the selection drop-down (and adding a "Simple Metal 1403 - Ref F" entry for the latest 1403) would prevent others from stepping in the same ditch.

-=dave

**Comment by PxT** [comment]
Fixed with commit 03e645e492f2003aef53a541d3f3cf343b344fc0
Please try the binary at https://dl.dropboxusercontent.com/u/68081994/printrbot/Printrbot%20Firmware%20Updater-3.0.zip and let me know if any issues.
